src/pages/EventPage/components/MetaInfo/MetaInfo.tsx

```diff
@@ -6,13 +6,14 @@ import { Event } from 'src/types/domain';
 
 type MetaInfoProps = Pick<
   Event,
-  'name' | 'startTime' | 'endTime' | 'location' | 'owner'
+  'name' | 'startTime' | 'endTime' | 'location' | 'owner' | 'imageUrl'
 >;
 
 function MetaInfo({
   name,
   startTime,
   endTime,
+  imageUrl,
```

Comment:
Is there any point in adding this if we're not using it yet?

Reply:
No